backend/controllers/userDetails.js

```diff
@@ -9,15 +9,17 @@ const userDetailsFactory = (
   reactUrl,
   manageUrl,
   maintainTitle,
+  showExtraUserDetails,
```

Comment:
page is now shared between external and DPS users, so need to toggle bits of the page off for now